scripts/globals/functional.lua

```diff
@@ -0,0 +1,95 @@
+-----------------------------------
+--
+-- Functional
+--
+-- Functional methods provide a means to simplify logic that consists in
+-- simple operations when iterating a table.
+-- In general, they can make code much more concise and readable, but they
+-- can also end up making it a cluttered mess, so use your judgement
+-- when deciding if you want to use these methods
+-----------------------------------
+
+fn = {}
```

Comment:
How did utils.lua get checked in? I am doing what they are on that other file and still have lua checks complain :/

Reply:
Global objects are bad and the linter looks out for them. The only exceptions are luautils functions and the xi object

Reply:
Removed.